Features/Assets/Sources/Scenes/AddTokenScene.swift

```diff
@@ -32,7 +32,7 @@ public struct AddTokenScene: View {
             Spacer()
             StateButton(
                 text: model.actionButtonTitle,
-                type: .primary(model.state),
+                type: .adoptiveGlassEffect(model.state),
```

Comment:
```suggestion
                type: .primary(model.state),
```

we should make this primary, the rest happens behind the scenes